### ISSUE TITLE
Create accessibility documentaion for JAWS, NVDA and other screen readers and accessibility tools for TB115

### ISSUE THREAD

**Issue by rtanglao** [thundernest/knowledgebase-issues#19]
Some relevant bugs:
* [1843486](https://bugzilla.mozilla.org/show_bug.cgi?id=1843486): NVDA table navigation with Ctrl+Alt+four arrow keys no longer available in message list
* [1846937](https://bugzilla.mozilla.org/show_bug.cgi?id=1846937): Message list doesn't scroll downwards far enough when using down arrow key with increased font
* [1844151](https://bugzilla.mozilla.org/show_bug.cgi?id=1844151): Threaded view: Statuses "Expanded" or "Collapsed" are announced at the end by the screen reader
* [1835530](https://bugzilla.mozilla.org/show_bug.cgi?id=1835530): "About" dialog is not readable or very cumbersome for screen readers

**Comment by emitche** [comment]
Current documentation that can be updated and expanded upon:

- [Thunderbird Accessibility Features](https://support.mozilla.org/en-US/kb/thunderbird-accessibility-features)
- [Keyboard shortcuts - perform common Thunderbird tasks quickly](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-thunderbird)

**Comment by rtanglao** [comment]
> Current documentation that can be updated and expanded upon: 
>     * [Thunderbird Accessibility Features](https://support.mozilla.org/en-US/kb/thunderbird-accessibility-features)
>     * [Keyboard shortcuts - perform common Thunderbird tasks quickly](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-thunderbird)
Thanks for finding these! As always, if there's specific edits you want to make them, please make them in SUMO. I am always fine with editing/revising any SUMO drafts. if you're not sure and just have an outline or some thoughts share them here or create a Google doc or wherever is easiest/best for you because i an also fine with editing/revising Google docs or whatnot! tl;dr we are flexible!

**Comment by rtanglao** [comment]
A to-do from Elisabeth on matrix
>With 115+, users can use the keyboard to manage the columns in the message list table view.

>Ordering columns: In 115, on Windows you can use Alt + Left and Alt + Right to move the columns in Table View. I think this is the same keys for Linux. For macOS it's Option + L and Option + R to move the columns.
This doesn't fully make sense yet, but to add/remove columns:

>To remove and add columns, tab into the message list column headers. Use the left/right arrow key until you get to Select columns to display. Hit Enter. Then using the up and down keys you can check off columns you want or do not want. Use Enter to select/deselect columns.

>This is from this thread: https://sunny.garden/@weirdwriter@tweesecake.social/111284773979065066.

**Comment by rtanglao** [comment]
@emitche sorry for the delay but i can't get Option + Left Arrow / Right Arraw to work on the Mac to move the columns. Alt + Left Arrow / Right Arrow works on Windows and Linux though! Any ideas on what works for Mac?

**Comment by rtanglao** [comment]
UPDATE: on macOS i have to tab to the columns after focusing on the Quick Filter search to make it work. On Windows and Linux, clicking on a column and tabbing from Quick filter works; clicking on a column in macOS doesn't work!

**Comment by rtanglao** [comment]
* Per: `Not being able to click on the column and then use the keyboard to move it is probably a bug.`, filed: bug [1862272](https://bugzilla.mozilla.org/show_bug.cgi?id=1862272): "macOS: Message List Columns can't be reordered with the keyboard after clicking on the column name in table mode. Have to hit tab. Works fine on macOS and Linux."

**Comment by rtanglao** [comment]
* for macOS:
![image](https://github.com/thunderbird/knowledgebase-issues/assets/45195/21ee6d2e-1ed7-4344-a91d-52539f7574a0)
* for Windows and Linux:
![image](https://github.com/thunderbird/knowledgebase-issues/assets/45195/04d4f23a-fa36-4012-a39a-9a69b14543dc)

**Comment by emitche** [comment]
> * Added in revision [269199](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-thunderbird/revision/269199)
> 
>     * for macOS:
>       ![image](https://user-images.githubusercontent.com/45195/280046364-21ee6d2e-1ed7-4344-a91d-52539f7574a0.png)
> 
>     * for Windows and Linux:
>       ![image](https://user-images.githubusercontent.com/45195/280046685-04d4f23a-fa36-4012-a39a-9a69b14543dc.png)

For macOS, it does work with keyboard navigation, but it doesn't work from a click. I might say "not working from a click on macOS" or "not fully working on macOS" or something like that.

This looks good. Thank you so much!

**Comment by rtanglao** [comment]
* changed it to ` "not fully working on macOS" ` https://support.mozilla.org/en-US/kb/keyboard-shortcuts-thunderbird/revision/269882
